book/website/reproducible-research/rdm/rdm-elns.md

```diff
@@ -0,0 +1,173 @@
+Mendeley for example began [encrypting their local database](https://www.zotero.org/support/kb/mendeley_import#mendeley_database_encryption) making it impossible to migrate your local library to different reference managers without going through their online library feature.
+This occured after they had reached a high degree of market penetration and been aquired by Elsevier.
+
+Many open tools on the other hand have very robust exort and import options, as they are not subject to this incentives trap, though this is far from garunteed.
+This might include a complete database export that would let you seemlessly migrate to a different deployment of the tool. 
+Or an archive of files organised by a for example a JSON file representation of your notes and files that retain most or all of the metadata and organisational information of your ELN.
+There remains the challenge when migrating of finding a tool that can import what your old tool exported, and retain most of all of its structure.
+This is where the development of standards for ELN formats could very valuable in the long run.
+
+This recent review {cite:ps}`higgins2022elns` provides a good overview of considerations when adopting an ELN solution. 
+It covers such things a regulatory compliance that is not yet touched on here.
+It does occasionally appear to conflate open-source solutions with self-hosted ones which need not necessarily be the case.
+Some companies will let you host proprietary apps on premises and conversely you can pay for 3rd party hosting and administration of open source applications.
+This is important as if you don't have the expertise or internal resources to administer a self-hosted instance of an open-source ELN solution you can still pay a 3rd party to do this for you. 
+In This case you get the benefits of professional support and the reassurance of an open solution.
+When using an open solution it is always important to consider how you can contribute to its ongoing development and maintenence, for example by donating to the project or if it makes sense given the makeup of your organisation and that of the project contributing developer time.
+You should still take regular local backups of exports from your hosting provider from which you could restore your ELN system with different hosting.
+This means that you retain the option to change providers as the hosting and support are no longer vertically integrated parts of the software as a service (SaaS) experience for you.
+
+There are many solutions to choose from: [How to pick an electronic laboratory notebook](https://doi.org/10.1038/d41586-018-05895-3) by {cite:ps}`Kwok2018eln` or [2019 Review of the Best Electronic Laboratory Notebooks](https://app.scientist.com/blog/2019/04/05/2019-review-of-the-best-electronic-laboratory-notebooks) may help you to find your solution!
```

Comment:
Maybe a bullet point overview of resources that provide an overview or considerations when choosing an ELN?